apps/organisasjon-orgnummer-service/src/main/java/no/nav/registre/orgnrservice/repository/OrgnummerRepository.java

```diff
@@ -0,0 +1,32 @@
+package no.nav.registre.orgnrservice.repository;
+
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+import no.nav.registre.orgnrservice.repository.model.OrgnummerModel;
+
+
+@Repository
+public interface OrgnummerRepository extends CrudRepository<OrgnummerModel, Long> {
+
+    List<OrgnummerModel> findAllByLedigIsTrue();
+    OrgnummerModel findByOrgnummer(String orgnummer);
```

Comment:
legg gjerne en blank linje imellom slik at man umiddelbart forstår at dette er to uavhengige metoder.